test/helpers.lua

```diff
@@ -257,11 +257,7 @@ function module.check_logs()
     table.concat(runtime_errors, ', ')))
 end
 
-function module.iswin()
-  return package.config:sub(1,1) == '\\'
```

Comment:
I think we should keep `iswin` as an alias for `is_os('win')` and then revert most of the changes to the tests. Then maybe add `isbsd` and `ismac`.

Reply:
This kinda doesn't make sense to me, as in my head the point of `is_os` was to have a single, small function as to prevent a bunch of helper functions like `isbsd` and `ismac` that does the same thing. Shouldn't we remove `is_os` then maybe?

Reply:
I'll leave it to you, I was just expressing my personal preference. But yeah, I'd probably remove `is_os` then.

Another option is to introduce a table so we can have `is_os.win` instead of `is_os('win')`.